lib/.env.rb

```diff
@@ -0,0 +1 @@
+ENV = {account_sid: "ACe2415a2472cdceefe771df528e659a25", auth_token: "6c294235c7b7d03d6a5546d5c2e51bb4", number: "441503505054"}
```

Comment:
Metrics/LineLength: Line is too long. [129/80]<br>Style/FileName: Use snake_case for source file names.